Public/Src/Cache/ContentStore/Distributed/NuCache/TransitioningContentLocationStore.cs

```diff
@@ -275,6 +285,11 @@ public Task<BoolResult> UnregisterAsync(Context context, IReadOnlyList<ContentHa
                     priorityQueue.Push(lastAccessTimes[i]);
                 }
 
+                //  Each page is guaranteed to have at least one entry.
+                first = page[0];

```

Comment:
nit: page.First(), page.Last()?

Reply:
Lance pointed out that Last() isn't optimized for List, so accessing through index is faster/better